reactor-core/src/main/java/reactor/core/publisher/Sinks.java

```diff
@@ -641,7 +675,11 @@ public Emission getReason() {
 		 * @deprecated to be removed shortly after 3.4.0-RC1. Use {@link #tryEmitError(Throwable)} and handle the result.
 		 */
 		@Deprecated
-		void emitError(Throwable error);
+		default void emitError(Throwable error) {
+			emitError(error, EmitFailureHandler.FAIL_FAST);
+		}
+
+		void emitError(Throwable error, EmitFailureHandler failureHandler);
```

Comment:
missing javadoc. insist on the fact that this should rely on `tryEmitError` (see https://github.com/reactor/reactor-core/pull/2405#discussion_r496589276)